### PR TITLE
DAG to index/archive landsat c3

### PR DESCRIPTION
**Pull request by sachitrajbhandari**

### CODE REVIEW

File: dags/k8s_index_ls_c3.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=5),
+    "aws_conn_id": "dea_public_data_upload",
+    "add_sqs_queue": "dea-dev-eks-landsat-c3-add",
+    "archive_sqs_queue": "dea-dev-eks-landsat-c3-archive",
+    "products": "ga_ls5t_ard_3 ga_ls7e_ard_3 ga_ls8c_ard_3",
+    "env_vars": {
+        # TODO: Change to dev DB credentials
+        "DB_HOSTNAME": "database-write.local",
```
Comment (NikitaGandhi) [inline dags/k8s_index_ls_c3.py:32]: please change DB_HOSTNAME -> db-writer

Comment (sachitrajbhandari) [reply dags/k8s_index_ls_c3.py]: Thanks. Changes done.

**Comment by alexgleith** [comment]
Hey @whatnick should this be updating the OWS and Explorer indexes too?

**Comment by sachitrajbhandari** [comment]
@alexgleith Review request - Indexer image tested locally and updated in DAG to test in dev env.

**Review by alexgleith — APPROVED** [review]
👍

**Comment by sachitrajbhandari** [comment]
@alexgleith @whatnick Merging is blocked for me. Has there been any changes in authorisation?

**Comment by alexgleith** [comment]
Can you please try now, @sachitrajbhandari

**Comment by sachitrajbhandari** [comment]
> Can you please try now, @sachitrajbhandari

No luck yet.

**Review by NikitaGandhi — APPROVED** [review]

**Pull request merged**